### ISSUE TITLE
Ignore characters in (parens) when querying the title

### ISSUE THREAD

**Issue by m-theredhead** [JasonHHouse/gaps#41]
Hello,

I am trying to run gaps 0.0.3 and it is failing to find many of my movies because the meta data in the file has data in it like 'Title (1080P)'.   So I get a query like:

2019-02-27 15:00:48.133 ERROR 19431 --- [           main] com.jasonhhouse.Gaps.GapsApplication     : Results not found for <title>  (1080P) (2011)

I know this is bad data in my files, but I have a large number of files with this in it.   Would it be possible to just ignore anything between () when searching for the title?

**Comment by JasonHHouse** [comment]
Are you running a branch or develop? I haven't merged file support yet, so it would only be plex returning values like that.

**Comment by m-theredhead** [comment]
I am running 0.0.3 branch.   It is plex that is returning those values as this is what the meta data in the file contains.   I was hoping that gaps could ignore those values in the title and send the query without the () data.

**Comment by JasonHHouse** [comment]
How are you setting that in the metadata for Plex? I'll need that to help build a repeatable example to remove.

**Comment by m-theredhead** [comment]
The files are almost all m4v.   Meta data in the files were created using MetaX which pulls from various sources.    The title was manually overwritten from: 'title' to 'title (resolution)' in the file metadata (not the file name).  When Plex imports the files, it appears to ignore the data in the title between the parens to match appropriately.  So in Plex, the files all match correctly.  I don't do it this way any more, but I sure don't want to fix 1000+ files

The easiest way reproduce is to take an existing file, edit with MetaX or some other metadata editor and add the date or anything in parens to the title.    Then import into plex.   It should still match the right movie, but if you look at the title, it will show the whole thing including the parens.  This then gives grief to Gaps as it doesn't match when it does the lookup.

**Comment by JasonHHouse** [comment]
Is this issue still happening?

**Comment by m-theredhead** [comment]
Sorry for the long delay.  It has taken me some time to get back to playing with Gaps.

I just did a completely fresh install using docker w/ gaps:latest and the issue persists.   

For example, it is saying 30 Days of Night and 30 Das of Night Dark Days are missing from the 30 Days of Night Collection, but they are both there - they just also have (1080p) in the title.

**Comment by m-theredhead** [comment]
Hello.    I spent a little more time on this issue this week and now I am even more confused.

I have updated to v0.2.2

In plex, I also figured out how to get it to not utilize the local metadata that is in the files.   It is now pulling data only from the moviedb.   As a result, of all my meta data with (1080p) in the movie title is now gone.   

I was hopeful this would resolve the issue with gaps, but unfortunately, it is still the same.   I found the recommendedMovies.json file and looking through it, many (most) of the movies it is recommending are movies already in my collection.   Any thoughts on how to further diagnose?
